src/patterns/national-insurance-numbers/index.md.njk

```diff
@@ -36,6 +36,40 @@ You should:
 
 {{ example({group: "patterns", item: "national-insurance-numbers", example: "default", html: true, nunjucks: true, open: true, size: "s"}) }}
 
+### Error messages
+
+Error messages should be styled like this:
+
+{{ example({group: "patterns", item: "national-insurance-numbers", example: "error", html: true, nunjucks: true, open: true, size: "s"}) }}
+
+Make sure errors follow the guidance in [error message](/components/error-message/) and have specific error messages for specific error states.
+
+#### If the National Insurance number is not in the right format and there is no example
```

Comment:
This is a good point. @stevenaproctor - what are your thoughts on this?

Reply:
'valid' is not good error message practice because it does not explain why the thing is not valid.

For example, BG123456C is not a valid National Insurance **because** it starts with BG. Without giving people this information, the valid does not mean anything. You have to explain why. These checks can be done in the frontend.

I think dropping 'real' from the error messages is a good idea because these checks will just make sure the number could be real.

The new messages are part of a PR. See https://github.com/alphagov/govuk-design-system/pull/537